### ISSUE TITLE
Use stretch setting in flexboxes

### ISSUE THREAD

**Issue by florianfesti** [florianfesti/boxes#32]
While we have a parameter in the flex settings it is often not used. May be the flex edge should use is directly. But this would require an opposing edge that does the same thing.

**Comment by MAKOMO** [comment]
Maybe a start could be to add the line

`c4 = c4 / self.edges["X"].settings.stretch`

also to the definitions of surroundingWall (the same way as in that definition in __init__.py) in the flexbox generators!?

**Comment by florianfesti** [comment]
Yes, the coding part is pretty trivial. This is more a question whether the generators still work after the change and whether the defaults are sane.

**Comment by matkoniecz** [comment]
If it is not used and flexboxes work, maybe set default to 1 (what should have not effect if my guess that it is multiplier is right)

That would make it far less confusing, I am trying to debug why my first box is a failure :)
